doc/router_certificates.md

```diff
@@ -0,0 +1,31 @@
+# Router certificates
+
+The default KubeCF installation generates a `router_ca` and a `router_ssl` variables. The former
+is the CA certificate that signs the later, which in turn is used in the gorouter for TLS
+termination.
+
+For production systems, where certificates signed by a well-known Certificate Authority is required,
```

Comment:
```suggestion
For production systems, where certificates signed by a well-known certificate authority is required,
```

While it's jargon, it's not a proper noun, nor a term we're defining in this legal document.